Makefile.am

```diff
@@ -0,0 +1,91 @@
+# $Id: Makefile.am,v 1.0.0-0 2016/05/06 14:24:14 Cnangel Exp $
```

Comment:
Any chance at all you could fold this into the existing Makefile OR Cmake (currently for windows but not an issue to retrofit) if you need some Autoconf functionality